payments/utils.py

```diff
@@ -364,7 +364,10 @@ def send_payment_notification(order, request, email=None):
     from payments.providers import get_payment_provider
 
     if email is None:
-        email = order.customer_email
+        if order_email := order.customer_email:
+            email = order_email
+        else:
+            raise ValidationError(_("Missing customer email"))
```

Comment:
I just added a similar check in another PR, when you rebase you can use this. When SMS sending is added this check needs to be modified again.